### PR TITLE
add pod context security

### PR DESCRIPTION
**Pull request by pierremahot**
add pod context security and set README to the real info
https://github.com/pmint93/helm-charts/issues/111

### CODE REVIEW

File: charts/metabase/templates/deployment.yaml [hunk 1]
```diff
@@ -311,6 +311,9 @@ spec:
         {{- if .Values.sidecars }}
         {{ toYaml .Values.sidecars | nindent 8 }}
         {{- end }}
+    {{- if .Values.podSecurityContext}}
+      securityContext:
+    {{- .Values.securityContext | toYaml | nindent 8 }}
```
Comment (pmint93) [inline charts/metabase/templates/deployment.yaml:316]: Shouldn't it be `.Values.podSecurityContext` ?

**Review by pmint93 — COMMENTED** [review]
Please also bump the chart version in Chart.yaml

File: charts/metabase/templates/deployment.yaml (hunk 1 above)
Comment (pmint93) [reply charts/metabase/templates/deployment.yaml:316]: @pierremahot

Comment (pierremahot) [reply charts/metabase/templates/deployment.yaml:316]: No is at the pod level and not the container level but it's called the same and it's not the same type of objects refert to the lube api full doc https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.31/#podspec-v1-core

Comment (pierremahot) [reply charts/metabase/templates/deployment.yaml:316]: My bad

File: charts/metabase/templates/deployment.yaml [hunk 2]
```diff
@@ -311,6 +311,9 @@ spec:
         {{- if .Values.sidecars }}
         {{ toYaml .Values.sidecars | nindent 8 }}
         {{- end }}
+    {{- if .Values.podSecurityContext}}
+      securityContext:
+    {{- .Values.podSecurityContext | toYaml | nindent 8 }}
```
Comment (pmint93) [inline charts/metabase/templates/deployment.yaml:316]: @pierremahot it seem we're missing `{{ - end }}` here

**Review by pmint93 — CHANGES_REQUESTED** [review]

File: charts/metabase/templates/deployment.yaml (hunk 2 above)
Comment (pierremahot) [reply charts/metabase/templates/deployment.yaml:316]: I have added the missing  `{{- end }}`

**Review by pmint93 — APPROVED** [review]

**Comment by pmint93** [comment]
Thanks for contributing !

**Pull request merged**